modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -81,6 +95,12 @@ From the command prompt, add [systemitem]``rhel-x86_64-server-6`` and [systemite
 From the {webui}, add [systemitem]``RHEL7 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES7 x86_64``.
 +
 From the command prompt, add [systemitem]``rhel-x86_64-server-7`` and [systemitem]``res7-suse-manager-tools-x86_64``.
+* For {es} 8:
++
+From the {webui}, add [systemitem]``RHEL8 Base x86_64`` and [systemitem]``SUSE Linux Enterprise Client Tools RES8 x86_64``.
```

Comment:
Just in case, go to ref40 and make sure the names are those.

Reply:
We don't have any RH channels in ref40, I just checked.

Reply:
We don't have then synced, but you could check at the setup wizard, the product list:

![image](https://user-images.githubusercontent.com/4226070/71087377-5309fe00-219c-11ea-91b2-bb2fdb263610.png)

This PR is still incorrect.

Reply:
![image](https://user-images.githubusercontent.com/4226070/71087638-f6f3a980-219c-11ea-95f5-991ece3e29cc.png)

Reply:
Do not include `(BETA)`, that's going away with this release.

Reply:
The second screenshot comes from clicking on the "list" icon on the right side of the channel row of the first screenshot.